grafana-plugin/src/components/Integrations/IntegrationBlock.tsx

```diff
@@ -24,22 +24,24 @@ export const IntegrationBlock: React.FC<IntegrationBlockProps> = ({
   className,
   toggle = noop,
 }) => {
+  const styles = useStyles2(getIntegrationBlockStyles);
+
   return (
-    <div className={cx('integrationBlock', className)}>
+    <div className={cx(className)}>
       {heading && (
         <Block
           bordered
           shadowed
-          className={cx('integrationBlock__heading', {
-            'integrationBlock__heading--noBorderBottom': !noContent,
+          className={cx(styles.integrationBlockHeading, {
+            [bem(styles.integrationBlockHeading, 'noBorderBottom')]: !noContent,
```

Comment:
```
 className={cx(styles.integrationBlockHeading, {
            [bem(styles.integrationBlockHeading, 'noBorderBottom')]: !noContent
})}
```
looks ease o make a typos, how does grafana core deal with nested selectors? I'm pretty sure it does not use bem.

And in general styling looks overcomplicated, to style something we need too much stuff to import

```
import { cx } from '@emotion/css';
import { useStyles2 } from '@grafana/ui';
import { bem } from 'utils/utils';
import { getIntegrationBlockStyles } from './IntegrationBlock.styles';
```

Can we remove at least one thing? the most suitable candidate seems to be a bem.

Reply:
Grafana doesn't use BEM pattern. Why remove any of it? I don't understand your reasons...

`cx` is for  composition
`useStyles2` is mandatory ...
`bem` is too build line blocks following BEM pattern so that you don't need to repeat the block, but instead rely on a helper that does that. BEM is very useful because it clearly expresses what the element modifier is supposed to do.
`getIntegrationBlockStyles` is outside the file so that we don't have too much code in the main file

I strongly recommend reading on https://getbem.com/